licenses/Khronos

```diff
@@ -0,0 +1,26 @@
+Copyright (c) 2015-2018 The Khronos Group Inc.
+
+Permission is hereby granted, free of charge, to any person obtaining a
+copy of this software and/or associated documentation files (the
+"Materials"), to deal in the Materials without restriction, including
+without limitation the rights to use, copy, modify, merge, publish,
+distribute, sublicense, and/or sell copies of the Materials, and to
+permit persons to whom the Materials are furnished to do so, subject to
+the following conditions:
+
+The above copyright notice and this permission notice shall be included
+in all copies or substantial portions of the Materials.
+
+MODIFICATIONS TO THIS FILE MAY MEAN IT NO LONGER ACCURATELY REFLECTS
```

Comment:
@gentoo/licenses, this clause seems to be the only 'special' thing about the license. Do we need an extra file for it?

Reply:
IMHO this is close enough to "MIT".

But another issue (with my QA hat on): Uppercase characters in package names are strongly discouraged, so presumably these should be named dev-util/spirv-headers and dev-util/spirv-tools (all lowercase).

Policy reference: https://devmanual.gentoo.org/ebuild-writing/file-format/index.html#file-naming-rules

Reply:
thank you for the feedback. license removed, packages renamed